Todo_CLI/todoCLIWithStretch.js

```diff
@@ -0,0 +1,107 @@
+///////////////////////////////////////////////////////////////////////////////////
```

Comment:
Do not produce duplicate code just for an additional feature. Stretch should be done in one file.

Reply:
ok, thanks